src/views/Dashboard.vue

```diff
@@ -231,7 +231,7 @@ export default {
   computed: {
     ...mapGetters(['isBenBcCccUlc', 'isHistorical', 'isRoleStaff', 'isCoaPending', 'getCoaEffectiveDate',
       'isAppTask', 'isAppFiling', 'getParties', 'isFirm', 'isSoleProp', 'isPartnership', 'getIdentifier',
-      'hasMissingInfoWarning', 'hasComplianceWarning', 'isAdminFreeze']),
+      'hasMissingInfoWarning', 'hasComplianceWarning', 'isAdminFreeze', 'getEditUrl']),
```

Comment:
I think there's 2 place in this component where you have to change editUrl -> getEditUrl.

Reply:
Good catch, I'll fix it.